mobile_api_guidance.md

```diff
@@ -0,0 +1,7 @@
+# Versioning
+
+Functionality provided by an API inevitably changes over time, and it’s sometimes necessary to make breaking, backwards-incompatible changes. In contrast to web app users, mobile app users must upgrade their software to take advantage of new functionality, and there is a period of overlap and adoption where two versions of a client are in circulation. It is beneficial to think about upgrade strategies early in your API’s development, and there are a few different approaches.
+
+The most common approach is to prefix URIs with a version, e.g. ‘/v1/method’. This allows clients that are configured to use a particular version of an API to continue using it after a new version of the API is released with a different prefix. Facebook is an example of an API that is versioned this way.
+
```

Comment:
I think a number of recent projects have used the HTTP Accept header for versioning, for example, 
```
GET /method HTTP/1.1

Accept: application/vnd.some-app.com; version=1
Host: some-app.com
...
etc
```